components/Quiz.tsx

```diff
@@ -29,7 +29,7 @@ export default function Quiz() {
   if (error) return <p>Oh no... {error.message}</p>;
 
   return (
-    <div className="p-10 mx-auto w-1/2 bg-slate-800 border-2 border-slate-700 rounded-lg">
+    <div className="py-10 px-5 sm:p-10 mx-auto w-1/2 bg-slate-800 border-2 border-slate-700 rounded-lg">
```

Comment:
```suggestion
    <div className="py-10 px-5 sm:p-10 mx-auto w-5/6 sm:w-1/2 bg-slate-800 border-2 border-slate-700 rounded-lg">
```

Reply:
@jSunpayco don't forget about this one too

Reply:
Ah sure, that definitely looks better